.lintrunner.toml

```
@@ -242,6 +242,7 @@ code = 'CLANGTIDY'
 include_patterns = [
     'aten/src/ATen/core/*.cpp',
     'c10/**/*.cpp',
+    'c10/core/**/*.h',
```

```suggestion
    'c10/core/**/*.h*',
```
in case someone ever puts hpp files in there.

---

No hpp in PyTorch

---

There may be someone using hpp in the future. I will change the pattern in the next patch